examples/deploy-aws-lambda/package.json

```
@@ -1,16 +1,15 @@
 {
-  "name": "bolt-js-aws-lambda",
+  "name": "bolt-js-aws-lambda-receiver",
   "version": "1.0.0",
-  "description": "Deploying to AWS Lambda ⚡️ Bolt for JavaScript",
+  "description": "Deploying to AWS Lambda ⚡️ Bolt for JavaScript using the AWSLambdaReceiver",
```

Update the receiver name to be consistent through the doc:
```suggestion
  "description": "Deploying to AWS Lambda ⚡️ Bolt for JavaScript using the AwsLambdaReceiver",
```